UKHomeOffice/cop-ui
# COP-8497 Exclude targets from team list

## AC
Do not show `action a target` tasks in the `group/team task list` as they are shown in `targets`

## Updated
Added a check for task type to the GET tasks calls to exclude `action a target` type taskNames.
Moved names to constants

## To test

- Your tasks
- - claim a target from /targets
- - go to your tasks
> you should see your 'action a target' task as well as your other tasks
- - search for 'action a target'
> your action a target task should appear in the results
- - search for 'record' (or any other word in task names that you have)
> you should see tasks related to your search

- Group tasks
- - go to team tasks
> you should not see any 'action a target' tasks
- - search for 'action a target'
> you should get no results
- - search for 'record' (or any other word in task names that you have)
> you should see tasks related to your search
